src/backends/pyplot.jl

```diff
@@ -173,6 +177,17 @@ end
 py_fillstyle(::Nothing) = nothing
 py_fillstyle(fillstyle::Symbol) = string(fillstyle)
 
+function py_get_matching_math_font(parent_fontfamily)
+    math_font_family = if parent_fontfamily == "sans-serif"
+        "dejavusans"
```

Comment:
What if we want to use a font other than `DejaVu` ?

Reply:
Only a few math fonts are supported (as opposed to normal fonts). Its only natural that selection of math fonts is limited

Reply:
Thuogh, if you use something "computer modern" it will try to set it as both
it is limited to "dejavu both serif and sans", stix, cm for math fonts

Reply:
Can you make it configurable via [`extra_kwargs`](http://docs.juliaplots.org/latest/input_data/#extra_kwargs) ?